qa/standalone/ceph-helpers.sh

```diff
@@ -172,12 +172,10 @@ function teardown() {
     local pattern="$(sysctl -n $KERNCORE)"
     # See if we have apport core handling
     if [ "${pattern:0:1}" = "|" ]; then
-      # TODO: Where can we get the dumps?
-      # Not sure where the dumps really are so this will look in the CWD
-      pattern=""
-    fi
+      echo "Misconfigured test environment, systemd-coredump not supported"
```

Comment:
i logged on to a random smithi xenial test node runing test. and ran
```console
kchai@smithi109:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
$ sysctl -n kernel.core_pattern
/home/ubuntu/cephtest/archive/coredump/%t.%p.core
```
so it seems that `kernel.core_pattern` was set as expected.